src/__tests__/lib/queries/getAccounts.test.ts

```diff
@@ -59,7 +59,6 @@ describe('getAccounts', () => {
     const accounts = await getAccounts();
 
     expect(accounts.root.guid).toEqual('a');
-    expect(accounts.a.guid).toEqual('a');
```

Comment:
Related to the change to stop showing two ROOT accounts in the selector (The `template` one shows)